src/core/src/pass/convert_precision.cpp

```diff
@@ -332,6 +332,8 @@ bool fuse_type_to_parameter(const std::shared_ptr<ngraph::Node>& node, element::
 
 bool fuse_type_to_convert(const std::shared_ptr<ngraph::Node>& node, element::Type to, size_t idx) {
     if (auto convert = ov::as_type_ptr<opset4::Convert>(node)) {
+        convert->get_rt_info()["precision"] =
```

Comment:
Why do we need to add precision to RT info?

Reply:
This precision is required for value range truncation.
For instance, if the Convert Layer Float->Bool is changed to Float->u8, this means that the Float-> Bool->u8 transformation chain should be done on the plugin side.

Reply:
Removed, but I disabled BOOL conversion tests.